### ISSUE TITLE
[Accessibilty]A11y_PowershellEditorServices_Installation_Keyboard: Tooltip of "Details,Contributions,Changelog,Dependencies" is not accessible to the keyboard users.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61709]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10
MAS1.1.1,MAS2.1.1
**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Navigate through keyboard such as "Tab" key.
4.Observed that whether tooltip of "Details,Contributions,Changelog,Dependencies" is accessible.

**Actual:**
Tooltip of "Details,Contributions,Changelog,Dependencies" is not accessible to the keyboard users.

**Expected:**
Tooltip of "Details,Contributions,Changelog,Dependencies" should accessible to the keyboard users.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User will not able to know the extra information of the control.

**MAS Reference:**
MAS2.1.1
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:**
[2730593_A11y_PowershellEditorServices_Installation_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2510372/2730593_A11y_PowershellEditorServices_Installation_Keyboard.pptx)
